### DISCUSSION TITLE
how to use --abs-proxy-base-path ?

### DISCUSSION THREAD

**Discussion by GitHub4LP** [coder/code-server#7010] [Q&A] [answered]
how to use --abs-proxy-base-path ?

**Comment by code-asher** [comment, marked as answer]
https://github.com/coder/code-server/blob/main/docs/guide.md#prefixing-absproxyport-with-a-path

Let me know if you run into trouble with the docs, happy to change the text if things are not clear.

**Reply by GitHub4LP** [reply]
Thanks, I was beginning to think I could do with one less layer: nginx or something.

**Reply by code-asher** [reply]
Ahhh nope, probably always a good idea to use a purpose-built reverse proxy anyway :D

**Reply by kimboslice99** [reply]
is --abs-proxy-base-path configurable from file or only cli? It is not made clear how to configure this.

installed via debian .deb package, for context.

**Reply by code-asher** [reply]
You should be able to configure it in the settings file or as a command line argument, same as any other flag.

**Comment by yuzhuoheng** [comment]
abs-proxy-base-path   It didn't work. This is my startup command:
![企业微信截图_17288752683468](https://github.com/user-attachments/assets/89b1e5e0-726b-4b52-b572-8af7436bc9be)
I tried to open it with a browser, and the page prompt was as follows:
![企业微信截图_1728875413984](https://github.com/user-attachments/assets/cd271829-8589-4194-8425-e28be1903222)

**Reply by code-asher** [reply]
I answered in a couple other places, please let me know if you have suggestions on how we can improve the documentation to make this less confusing.  Thank you!
